File: udemy/ds-and-a/csharp-solutions/DSAndA.Test/LinkedLists/Assignment2/Reverse.Test.cs
using DSAndA.LinkedLists;
using Xunit;

namespace DSAndA.Test.LinkedLists.Assignment2
{
    public class ReverseTestFixture
    {
        [Fact]
        public void ReverseTest1()
        {
            LinkedList<int> linkedList = new LinkedList<int>();
            linkedList.InsertAtHead(2);
            linkedList.InsertAtHead(1);

            linkedList.Reverse();

            Assert.Equal(expected: "2,1,", actual: linkedList.ToString());
        }
    }
}

File: udemy/ds-and-a/javascript-solutions/src/tests/LinkedLists/Assignment2/Reverse.test.js
const LinkedList = require("../../../LinkedLists/LinkedList");

test('Reverse order of elements in linked list', () => {
    let linkedList = new LinkedList();
    linkedList.insertAtHead(2);
    linkedList.insertAtHead(1);
    linkedList.insertAtHead(0);

    expect(linkedList.toString()).toBe('0, 1, 2');

    linkedList.reverse();

    expect(linkedList.toString()).toBe('2, 1, 0');
});